inferlo/pairwise/optimization/map_lp_test.py

```diff
@@ -0,0 +1,74 @@
+import numpy as np
+
+from inferlo import PairWiseFiniteModel
+from inferlo.pairwise.optimization.map_lp import map_lp
+from inferlo.pairwise.testing import grid_potts_model, tree_potts_model, \
```

Comment:
This should be inferlo.testing - that's why tests fail.
Maybe you have stale local version of repository - you should make sure it's up-to-date with what is repository. For that you should use git pull and git merge master.